src/main/java/com/inorg/services/product/service/CustomerServiceImpl.java

```diff
@@ -131,4 +126,73 @@ public CustomerPagedQueryResponse getCustomerByShoeSize(String preferredShoeSize
                 .executeBlocking()
                 .getBody();
     }
+    @Override
+    public CustomerToken createEmailVerificationToken(String customerId){
+        return apiRoot.customers()
+                .emailToken()
+                .post(CustomerCreateEmailTokenBuilder.of()
+                      .id(customerId)
+                      .ttlMinutes(60L)
+                      .build())
+                .executeBlocking()
+                .getBody();
+    }
+
+    @Override
+    public Customer verifyEmailToken(String token){
```

Comment:
The method Should be getCUstomerByEmailTOken reference: https://docs.commercetools.com/api/projects/customers#get-customer-by-email-token